### PR TITLE
Add new info command and update Re: 0.3.0 release

### PR DESCRIPTION
**Pull request by tpmccallum**
Content must go through a pre-merge checklist.

Relates to https://github.com/fermyon/cloud-plugin/pull/94

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors
- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/cloud/cloud-command-reference.md [hunk 1]
```diff
@@ -665,14 +937,73 @@ SUBCOMMANDS:
...
+
+{{ startTab "v0.1.2"}}
+
+Spin compatibility: `>= v1.3`
+
+<!-- @selectiveCpy -->
+```console
+$ spin cloud sqlite delete --help
+
+Delete a SQLite database
```
Comment (kate-goldenring) [inline content/cloud/cloud-command-reference.md:981]: Same as above
```suggestion
Delete a NoOps SQL database
```

File: content/cloud/cloud-command-reference.md [hunk 2]
```diff
@@ -807,6 +1139,37 @@ OPTIONS:
...
 
+{{ startTab "v0.3.0"}}
+
+Spin compatibility: `>= v1.3`
+
+<!-- @selectiveCpy -->
+```console
+$ spin cloud sqlite execute --help
+
+Execute SQL against a SQLite database
```
Comment (kate-goldenring) [inline content/cloud/cloud-command-reference.md:1150]:
```suggestion
Execute SQLite statements against a NoOps SQL database
```

File: content/cloud/cloud-command-reference.md [hunk 3]
```diff
@@ -866,6 +1229,33 @@ SUBCOMMANDS:
...
 
+{{ startTab "v0.3.0"}}
+
+Spin compatibility: `>= v1.3`
+
+<!-- @selectiveCpy -->
+```console
+$ spin cloud sqlite help    
+
+Manage Fermyon Cloud SQLite databases
```
Comment (kate-goldenring) [inline content/cloud/cloud-command-reference.md:1240]:
```suggestion
Manage Fermyon Cloud NoOps SQL databases
```

File: content/cloud/cloud-command-reference.md [hunk 4]
```diff
@@ -927,6 +1317,34 @@ OPTIONS:
...
 
+{{ startTab "v0.3.0"}}
+
+Spin compatibility: `>= v1.3`
+
+<!-- @selectiveCpy -->
+```console
+$ spin cloud sqlite list --help
+
+List all SQLite databases of a user
```
Comment (kate-goldenring) [inline content/cloud/cloud-command-reference.md:1328]:
```suggestion
List all NoOps SQL databases of a user
```

**Review by kate-goldenring — CHANGES_REQUESTED** [review]

**Review by macolso — APPROVED** [review]
LGTM

File: content/cloud/cloud-command-reference.md (hunk 1 above)
Comment (kate-goldenring) [reply content/cloud/cloud-command-reference.md:981]: @macolso we are still missing this change i believe

**Review by kate-goldenring — APPROVED** [review]

**Pull request merged**